data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/PostAttesterDuties.java

```diff
@@ -110,11 +108,12 @@ public void handle(Context ctx) throws Exception {
       final UInt64 epoch = UInt64.valueOf(parameters.get(EPOCH));
       final UInt64[] indexes = jsonProvider.jsonToObject(ctx.body(), UInt64[].class);
 
-      SafeFuture<Optional<List<AttesterDuty>>> future =
+      SafeFuture<Optional<GetAttesterDutiesResponse>> future =
```

Comment:
nit: rename to `PostAttesterDutiesResponse`

Reply:
Done.